### PR TITLE
RenameFreestyleProjectRDVTest

### PR DESCRIPTION
**Pull request by DVRudoi**
US003 - RenameFreestyleProjectRDVTest

### CODE REVIEW

File: src/test/java/RenameFreestyleProjectRDVTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class RenameFreestyleProjectRDVTest extends BaseTest {
+    public void deleteItem() {
+        WebDriverWait wait = new WebDriverWait(getDriver(), 10);
```
Comment (GitForProjects) [inline src/test/java/RenameFreestyleProjectRDVTest.java:11]: we already have BaseTest method - getWait5() or getWait20()
I think you can use one of them

Comment (DVRudoi) [reply src/test/java/RenameFreestyleProjectRDVTest.java]: Сorrected

**Review by GitForProjects — APPROVED** [review]
looks good

**Review by GettingPro — APPROVED** [review]
LGTM

**Pull request merged**